src/web/components/ArticleContainer.tsx

```diff
@@ -17,6 +18,25 @@ const articleContainer = css`
 	z-index: 1;
 `;
 
+const carrotStyles = css`
```

Comment:
Very minor so feel free to ignore.
Would prefer `carrotAdStyles` to be more specific

Reply:
```suggestion
const carrotAdStyles = css`
```